README.md

```diff
@@ -80,13 +80,13 @@ Join by phone: +1 405-792-0633‬ PIN: ‪299 906‬#
 
 The easiest way to get etcd is to use one of the pre-built release binaries which are available for OSX, Linux, Windows, and Docker on the [release page][github-release].
 
-For more installation guides, please check out [play.etcd.io](http://play.etcd.io) and [operating etcd](https://github.com/etcd-io/etcd/tree/master/Documentation#operating-etcd-clusters).
+For more installation guides, please check out [play.etcd.io](http://play.etcd.io) and [operating etcd](https://etcd.io/docs/latest/op-guide).
```

Comment:
this op-guide link works but why the branch-management and dl-build doesn't? Thanks!